projects/developer/src/app/processing/jobs/component.html

```diff
@@ -90,6 +90,13 @@ <h2><i class="fa fa-cube"></i> Jobs <span *ngIf="count">({{ count }})</span></h2
                         </a>
                     </div>
                 </div>
+                <div *ngSwitchCase="'inputs'">
+                    <div *ngIf="rowData.input_files">
+                        <span [tooltipStyleClass]="'max-size'" [pTooltip]="makeTooltip(rowData.input_files)">

```

Comment:
Very minor, but can you just leave this without the binding? `<span tooltipStyleClass="max-size" ...`

Reply:
Done.